app/browser/injectedBrowser.js

```diff
@@ -0,0 +1,268 @@
+// Injected browser tools for Teams for Linux
+// This script runs in the browser context and initializes browser tools
+
+(function () {
+  console.log("=== Teams for Linux Browser Tools Loading ===");
+  
+  // Get the electronAPI that's available via contextBridge
+  if (!window.electronAPI) {
+    console.error("electronAPI not available - preload script may not have loaded");
+    return;
+  }
+
+  // Get config from main process
+  window.electronAPI.getConfig().then((config) => {
+    console.log("Browser tools config received");
+    
+    // Initialize ActivityManager for handling awayOnSystemIdle
+    initializeActivityManager(config);
+    
+    // Initialize MQTT status monitor if enabled
+    if (config.mqtt && config.mqtt.enabled) {
+      initializeMQTTStatusMonitor(config);
+    }
+    
+    // Initialize other browser tools
+    initializeOtherTools(config);
+  }).catch((error) => {
+    console.error("Failed to get config:", error);
+  });
+
+  function initializeActivityManager(config) {
```

Comment:
we shouldn't need to do this here if you move the injection logic to the preload script (that should have most of it already there) 

See https://github.com/IsmaelMartinez/teams-for-linux/blob/main/app/browser/preload.js#L170